File: src/main/java/com/dh/ondot/member/core/exception/TokenMissingException.java
package com.dh.ondot.member.core.exception;

import com.dh.ondot.core.exception.UnauthorizedException;

import static com.dh.ondot.core.exception.ErrorCode.TOKEN_MISSING;

public class TokenMissingException extends UnauthorizedException {

    public TokenMissingException() {
        super(TOKEN_MISSING.getMessage());
    }

    @Override
    public String getErrorCode() {
        return TOKEN_MISSING.name();
    }
}

File: src/main/kotlin/com/dh/ondot/member/core/exception/TokenMissingException.kt
package com.dh.ondot.member.core.exception

import com.dh.ondot.core.exception.ErrorCode.TOKEN_MISSING
import com.dh.ondot.core.exception.UnauthorizedException

class TokenMissingException :
    UnauthorizedException(TOKEN_MISSING.message) {
    override val errorCode: String get() = TOKEN_MISSING.name
}
